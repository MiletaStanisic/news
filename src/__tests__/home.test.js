import * as React from 'react';
import { shallow } from 'enzyme';
import Home from '../pages/home';

jest.mock('react-redux', () => ({
  useSelector: jest.fn(fn => fn()),
  useDispatch: jest.fn(fn => fn())
}));

describe('Home', () => {
  it('should render component', () => {
    const wrapper = shallow(<Home />);

    expect(wrapper).toMatchSnapshot();
  });
});
