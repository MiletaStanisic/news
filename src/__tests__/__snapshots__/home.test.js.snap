// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example should render component 1`] = `ShallowWrapper {}`;

exports[`Home should render cards 1`] = `ShallowWrapper {}`;

exports[`Home should render component 1`] = `ShallowWrapper {}`;
